release_notes.md

```diff
@@ -5,4 +5,5 @@
 - Change Description (#pr)
 
 **Release sprint:** Sprint 79
-[ [bugs](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+79%22+label%3Abug+is%3Aclosed) | [features](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+79%22+label%3Afeature+is%3Aclosed) ]
\ No newline at end of file
+[ [bugs](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+79%22+label%3Abug+is%3Aclosed) | [features](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+79%22+label%3Afeature+is%3Aclosed) ]
+[ [Python Worker 1.1.3](https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.3) | [Python Library 1.3.0](https://github.com/Azure/azure-functions-python-library/releases/tag/1.3.0) ]
```

Comment:
Do you intend for this to become part of the permanent template? It would make sense, just curious about the intention here.

Reply:
@fabiocav, oh, thanks for letting me know this is a permanent template. I thought this one is just a release note for each sprint, since it has the title "**Release sprint:** Sprint 79".

I will remove this Python worker release note from here.

Reply:
It is.... I'm just wondering if we want to add the current worker versions for every release as part of the template. If all you want is the mention there, just add an item to the bullet list above

Reply:
I think we should have a mention here, for what is worth

Reply:
OK